chatminer/chatparsers.py

```diff
@@ -48,11 +48,11 @@ def get_df(self):
 
 
 class Parser(ABC):
-    def __init__(self, filepath):
+    def __init__(self, filepath: str):
         self._file = Path(filepath)
         assert self._file.is_file()
 
-        self._raw_messages = []
+        self._raw_messages: Any = []
```

Comment:
can we be more restrictive here? Shouldn't we be able to use 
```suggestion
        self._raw_messages: List[Dict] = []
```

Reply:
The `json.load()` function returns `Any` so things like this:
```py
self._raw_messages = json.load(f)["messages"]
```
wouldn't work.